websites/docs-smoke-test/src/release-notes/bugfix.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Release notes not reappearing when changed
+date: 2020-03-16
+type: Resolved Issue
+topics:
+  - Settings
```

Comment:
Same for the topics. Provide only identifiers that are mapped to pre-defined labels.

Reply:
We came to the conclusion to not have predefined topics but make it dynamic, i.e. the UI is populated from the topics that are actually used.   Basically like most filter engines work - most painless for the authors to add more. 

We will only need separate configuration once the need comes up to hide old topics in the filter nav.  That would need to be a per-website data file defining the filterable topics, but even that would not be a hard whitelist (you never want to touch old release notes from five years ago).

Reply:
Sure, my point is that in the front matter you only provide the identifiers, to avoid typos etc.
Each website can then define its own list of topics as a config map.